O/OpenBLAS/common.jl

```diff
@@ -231,6 +231,6 @@ function openblas_dependencies(platforms; kwargs...)
         Dependency("CompilerSupportLibraries_jll"),
         HostBuildDependency("FlangClassic_jll"),
         BuildDependency("LLVMCompilerRT_jll"; platforms=filter(p -> sanitize(p)=="memory", platforms)),
-        BuildDependency("FlangClassic_RTLib_jll"; platforms=filter(p -> sanitize(p)=="memory", platforms))
+        Dependency("FlangClassic_RTLib_jll"; platforms=filter(p -> sanitize(p)=="memory", platforms))
```

Comment:
Something is wrong: this is being added to all x86_64-linux-gnu platforms, also where sanitize should be `!= "memory"`: https://github.com/JuliaBinaryWrappers/OpenBLAS_jll.jl/commit/45a51542ddafb16d4e999dce90a2f42de0d32968#diff-6f20eab6da0e9c8d8e793690f33f2566d327c5996b1e065186768d399cf9a3d4

Reply:
This endeavor is stressing BinaryBuilder out. I wonder what happened here.